app/models/carto_uploader.rb

```diff
@@ -1,28 +1,47 @@
+require 'csv'
 require 'uri'
 require 'net/http'
+require 'json'
 
 class CartoUploader
   CARTODB_ACCOUNT = ENV["CARTODB_ACCOUNT"]
   CARTODB_APIKEY = ENV["CARTODB_APIKEY"]
   CARTODB_URI_GET = "https://#{CARTODB_ACCOUNT}.cartodb.com/api/v2/sql?q=__SQLQUERY__&api_key=#{CARTODB_APIKEY}"
   CARTODB_URI_POST = "https://#{CARTODB_ACCOUNT}.cartodb.com/api/v2/sql"
-  CARTODB_TABLE = ENV["CARTODB_TABLE"]
-  
-  CARTODB_INSERT_MODE_INDIVIDUAL = "ind"
-  CARTODB_INSERT_MODE_MASS = "mass"
+  CARTODB_TABLE = ENV["CARTODB_TABLE"]  
+  CLASSIFICATIONS_PER_BATCH = 200
 
-  def initialize(mode = CARTODB_INSERT_MODE_MASS)
-    @mode = mode
+  def initialize()
   end
   
   def upload(data)
-    allVals = []  #For Mass Insertion mode only.
-    keys = []     #Declaration for Mass Insertion mode.
+    #Determine what's already in the database
+    #--------------------------------
+    latestClassificationID = getLatestClassificationID()
```

Comment:
Could you switch this (and others) over to `snake_case` which is the convention in Ruby?